### PR TITLE
TMI updates

### PR DESCRIPTION
**Pull request by jflorkey-donordrive**
some of the indenting in the compare looks strange, but the files look correct to me

### CODE REVIEW

File: resources/donations.md [hunk 1]
```diff
@@ -23,36 +23,63 @@ The response from the `donations` endpoint is an array of donation objects.
...
-    "amount": 20,
+    "amount": 10,
     "donorID": "EB8610A3FC435D58",
     "avatarImageURL": "https://static.donordrive.com/clients/try/img/avatar-constituent-default.gif",
     "createdDateUTC": "2019-10-30T18:01:18.513+0000",
     "eventID": 581,
     "teamID": 5074,
-    "donationID": "DF4E676D0828A8D5"
+    "donationID": "DF4E676D0828A8D5",
+	"links": {
```
Comment (timmixell) [inline resources/donations.md:33]: can you please check indentation on your new properties?

File: resources/participants.md [hunk 2]
```diff
@@ -42,22 +42,33 @@ The response from the `participants` endpoint is a single fundraiser object.
...
 |`isTeamCaptain`|`boolean`|`true` if this Participant is the captain of a team||`true`|Team Participants only|
 |`links`|`object`|An object containing related resources|||`donate`: The donation URL for this Participant<br />`page`: The URL for this Participant's Fundraising Page<br />`stream`: The URL for the Live Fundraising(TM) stream associated with this Participant|
+|`mustBeTeamCaptain`|`boolean`|`true` if the participant's type requires them to be a team captain|No|No||
 |`numAwardedBadges`|`integer`|The number of badges this Participant has received||Yes|Added: 1.2|
 |`numIncentives`|`integer`|The number of fundraising incentives this Participant has created||Yes|Added: 1.3|
 |`numMilestones`|`integer`|The number of fundraising milestones this Participant has created||Yes|Added: 1.3|
 |`numDonations`|`integer`|The number of donations this Participant has received|Yes|Yes||
 |`participantID`|`integer`|The unique ID of this Participant|Yes|Yes||
+|`participantTypeCode`|`string`|The code associated with the participant's type|Yes|Yes||
+|`streamingChannel`|`string`|The participant's streaming channel name|No|No||
```
Comment (timmixell) [inline resources/participants.md:67]: please note that we identify entities as proper nouns (ex: `Participant`)

File: resources/donations.md (hunk 1 above)
Comment (jflorkey-donordrive) [reply resources/donations.md]: it was tabs vs spaces. they're all fixed now

File: resources/participants.md (hunk 2 above)
Comment (jflorkey-donordrive) [reply resources/participants.md]: Gotcha. Think I got all the places where they're used that way.

**Review by timmixell — APPROVED** [review]

**Pull request merged**